src/scss/components/_cards.scss

```diff
@@ -78,6 +84,21 @@ $card-section-collapsible-paddingRight--aboveMid: 60px;
             }
         }
 
+        // used to create a section at the end of a card which has a background colour applied
+        .c-card-section--highlight {
+            background-color: $card-section-highlight-backgroundColor;
+            border-bottom-right-radius: $card-radius;
+            border-bottom-left-radius: $card-radius;
+            color: $card-section-highlight-color;
+            margin-top: spacing(x2);
+            padding: (spacing() / 2) spacing();
```

Comment:
Reckon it's worth adding a `0.5` spacing option?  Know this is being used in a few places now.

Reply:
Cool, done. Didn't know you specify keys like that 😄